### ISSUE TITLE
dice rolling appears to be broken

### ISSUE THREAD

**Issue by jkomg** [kakaroto/Beyond20#223]
I can't get any dicerolls to appear in roll20 at all, error is always: 

<img width="553" alt="Screen Shot 2020-04-27 at 6 03 02 AM" src="https://user-images.githubusercontent.com/1976048/80365274-ed660680-884c-11ea-80c1-a2f3b235315c.png">

using Chrome Version 81.0.4044.122 (Official Build) (64-bit)
OSX Catalina: 10.15.4 (19E287)

None of my players could see rolls either, all anyone saw was: 

<img width="301" alt="Screen Shot 2020-04-27 at 6 05 38 AM" src="https://user-images.githubusercontent.com/1976048/80365404-243c1c80-884d-11ea-9360-ceb375c30d08.png">

(with their own name, of course)

**Comment by jkomg** [comment]
Update: I am runnin two roll20 games, and the plugin appears to work in one and not in the other - I have no idea why. Same browser and OS in both tests.

**Comment by jkomg** [comment]
I created a new game, and the same issue is occurring. My old games seem to work fine. I ran both games at the same time in two browser windows, cleared the chat log of both and cleared the console log. Here is a fresh attack from a Kobold running off of dndbeyond: (apologies from the large screenshot)

<img width="3196" alt="Screen Shot 2020-04-27 at 6 18 39 AM" src="https://user-images.githubusercontent.com/1976048/80366493-fbb52200-884e-11ea-9623-4677df7464dc.png">

Is this operator error in some way?
